applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -105,6 +105,14 @@
 #endif /* CONFIG_AUDIO_SOURCE_USB */
 #endif /* CONFIG_TRANSPORT_BIS */
 
+#if (CONFIG_AUDIO_SAMPLE_RATE_48000_HZ)
```

Comment:
This should be combined with the changes in https://github.com/nrfconnect/sdk-nrf/pull/9953, let's see who merges first